templates/emails/thankyou.opensource.hbs

```diff
@@ -16,16 +12,16 @@ Subject: Thank you for your {{currency order.totalAmount currency=order.currency
 
 {{#if interval}}
   {{#if firstPayment }}
-    <p>We would like to thank you for your pledge to give {{currency order.totalAmount currency=order.currency}} per {{interval}}.
-      Your first payment was received on {{moment order.createdAt}}. As the collective's fiscal sponsor, Open Source Collective, a public recognized tax-exempt organization under IRS Section 501(c)(6), acknowledges your donation.</p>
+    <p>Thank you for contributing {{currency order.totalAmount currency=order.currency}} per {{interval}} to {{collective.name}}.
+       Your first payment was made on {{moment order.createdAt}}. As the collective's fiscal sponsor, Open Source Collective, a public recognized tax-exempt organization under IRS Section 501(c)(6), acknowledges your donation.</p>
```

Comment:
```suggestion
       Your first payment was made on {{moment order.createdAt}}. As the collective's fiscal sponsor, Open Source Collective acknowledges your donation.</p>
```
Remove confusing note about Open Source Collective's tax status as users are confused about the implications on their own tax affairs.